src/Tribe/Timezones.php

```diff
@@ -279,10 +279,29 @@ public static function to_utc( $datetime, $tzstring, $format = null ) {
 		$new_datetime = date_create( $datetime, $local );
 
 		if ( $new_datetime ) {
-			$new_datetime->setTimezone( $utc );
-			$format = ! empty( $format ) ? $format : Tribe__Date_Utils::DBDATETIMEFORMAT;
-
-			return $new_datetime->format( $format );
+			if ( empty ( $format ) ) {
+				/**
+				 * In order to avoid problems with dates that fall into DST for local timeszones we need to split into: date & time
+				 * doing this we convert to UTC each in order to avoid a DST conversion based on date and time.
+				 *
+				 * This is important specially for recurring events as some of them might fall inside or outside of DST periods and as the conversion
+				 * might fall into timezones with DST PHP will automatically convert an hour from normal to DST or viceversa
+				 * in most cases the event that was supposed to be started at 12:00pm should remain at the same time
+				 * regardless if is DST or not.
+				 */
+				$date = date_create( $new_datetime->format( Tribe__Date_Utils::DBDATEFORMAT ), $local );
```

Comment:
Does `date_create()` throw warnings when is not a good format?
If so, we want to wrap this on `try() catch`

Reply:
Right! I think we control the format but it might be wise to do it anyways. BTW do you know the error of the sniff above not sure what that really means because the value tested on empty is null maybe?

Reply:
That is a false positive, dunno why it does that.